### ISSUE TITLE
can't resize canvas

### ISSUE THREAD

**Issue by romaincorral** [patrimart/ngvas#2]
Thank for this module !
But I want to rezise the canvas when I receive windows resize event and it's impossible.
I success to change ngvas widht and height but I think the binding whith html5 canvas element don't work.
Have you a solution ?

**Comment by JoshDobbin** [comment]
I found the issue was that the _canvasGroup property type was not being updated when the size changed. It meant that while the canvas itself would change shape when the height/width were set, it wouldn't allow drawing outside of the initial canvas size.

Adding something like this to the code that resizes the canvas helped me:
```
this.certCanvas._canvasGroup = new CanvasGroup(this.certCanvas.canvasRef.nativeElement, undefined, this.certCanvas._isActive);
```

I think that re-initializing the _canvasGroup property on the setters for height and width would solve this better. I might submit a PR.

**Comment by JoshDobbin** [comment]
Added a PR #5 for updating canvas dimensions. Should remove the need for the code I added above.

**Comment by patrimart** [comment]
This fix is in v0.7.6. Thanks, Josh.

**Comment by willyguevara** [comment]
@patrimart nice component!

i have some questions:
1) how can i group two ngvas?
2) is this for resizing the canvas programatically?
